Microsoft.Toolkit.Uwp.UI/Cache/CacheBase.cs

```diff
@@ -53,6 +53,11 @@ protected CacheBase()
         }
 
         /// <summary>
+        /// Gets or sets a value indicating whether context should be maintained until type has been instantiated or not.
+        /// </summary>
+        public bool MaintainContext { get; set; }
```

Comment:
@ScottIsAFool if the idea of using this to control ConfigureAwait is acceptable then I'll change it to a field and push it in